rules-reviewed/eap7/eap6/resteasy.windup.xml

```diff
@@ -737,6 +737,93 @@
                 </hint>
             </perform>
         </rule>
+        <rule id="resteasy-eap6-000140">
+            <when>
+                <javaclass references="javax.ws.rs.Produces">
+                    <location>ANNOTATION</location>
+                    <annotation-literal name="value" pattern="{*}yaml"/>
+                </javaclass>
+            </when>
+            <perform>
+                <iteration>
+                    <when>
+                        <not>
+                            <file-exists filename="javax.ws.rs.ext.Providers"/>
+                        </not>
+                    </when> 
+                    <perform>
+                        <hint title="Resteasy Yaml Provider is deprecated and disabled by default" effort="3" category-id="mandatory">
+                            <message>The resteasy-yaml-provider module is not supported. 
+                            If you would like to use YAML Resteasy Provider even it is not recommended, 
+                            you need to add the SnakeYAML library (Maven dependency) into your application and enable it by creating file `META-INF/services/javax.ws.rs.ext.Providers` with line `org.jboss.resteasy.plugins.providers.YamlProvider` in that file.
+                            </message>
+                            <link title="RESTEasy YAML Provider Setting Changes"
+                                href="https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.1.beta/html-single/migration_guide/#migrate_resteasy_yaml_provider_setting_changes"/>
+                            <link title="RESTEasy YAML Provider"
+                                href="https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.0/html-single/developing_web_services_applications/#jaxrs_yaml_provider"/>
+                            <tag>resteasy</tag>
+                            <tag>yaml</tag>
+                        </hint>
+                    </perform>
+                </iteration>
+            </perform>
+        </rule>
+        <rule id="resteasy-eap6-000141">
+            <when>
+                <javaclass references="javax.ws.rs.Produces">
+                    <location>ANNOTATION</location>
+                    <annotation-literal name="value" pattern="application/x-java-serialized-object"/>
+                </javaclass>
```

Comment:
There's no condition for the `@Consumes(value="application/x-java-serialized-object")` case while it was enlisted in the JIRA [WINDUPRULE-316](https://issues.jboss.org/browse/WINDUPRULE-316).

Reply:
good catch, I will fix it